### PR TITLE
Please add my website

### PR DESCRIPTION
**Pull request by spianoDev**

### CODE REVIEW

File: sites/spiano.dev.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+---
+title: 'spiano.dev'
+url: 'https://www.spiano.dev/'
+tags: ['web developer', 'full stack', 'software engineer]
+nsfw: false
+rss: false
+---git
```
Comment (hankchizljaw) [inline sites/spiano.dev.md]: Whoops, you’ll need to just get rid of this `git` bit

File: sites/spiano.dev.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+---
+title: 'spiano.dev'
+url: 'https://www.spiano.dev/'
+tags: ['web developer', 'full stack', 'software engineer]
```
Comment (hankchizljaw) [inline sites/spiano.dev.md]: Just add the closing `'`, so it’s `'software engineer']`

Comment (spianoDev) [reply sites/spiano.dev.md]: All mistakes have been corrected in today's commit. Sorry about that!

Comment (hankchizljaw) [reply sites/spiano.dev.md]: Fabulous! No need to apologise—I normally don't bother checking and keep tanking the site 😂

**Comment by hankchizljaw** [comment]
Welcome aboard!

**Pull request merged**